documentation/messaging_app_developer/connecting.adoc

```diff
@@ -155,6 +155,29 @@ container.on('sendable', function (context) {
 container.connect({port:443, host:'<messaging-route-hostname>', transport:'tls', rejectUnauthorized:false});
 ----
 
+==== {ClientAmqpJavaScriptName} using WebSockets
+
+[source,javascript,options="nowrap"]
+----
+var container = require('rhea');
+var WebSocket = require('ws');
+var ws = client.websocket_connect(WebSocket);
+
+container.on('connection_open', function (context) {
+    context.connection.open_receiver('myqueue');
+    context.connection.open_sender('myqueue');
+});
+container.on('message', function (context) {
+    console.log(context.message.body);
+    context.connection.close();
+});
+container.on('sendable', function (context) {
+    context.sender.send({body:'Hello World!'});
+    context.sender.detach();
+});
+container.connect({username: 'user1', password: 'password1', transport:'tls', rejectUnauthorized:false, connection_details: ws("wss://<messaging route hostname>:443", ["binary"], {rejectUnauthorized: false})});
```

Comment:
The 'transport:'tls', rejectUnauthorized:false,' part will be ignored as connection_details is specified (and wss implies tls and rejectUnauthorized is specified as an option to the websocket).

Reply:
@grs Should be fixed now. Also I removed the username/password for now, as there isn't any provided for the other client examples.

Reply:
For the javascript examples we should leave in the username (e.g. as anonymous), as otherwise it will not use a sasl layer, which is required. Other languages should use sasl layer by default.